ngrok/README.md

```diff
@@ -0,0 +1,77 @@
+## Overview
+
+ngrok delivers instant ingress to your applications in any cloud, private network, or devices with authentication, load balancing, and other critical controls via our global points of presence.
+
+The ngrok platform includes a Datadog event destination integration.  When used in conjunction with ngrok HTTP events the visualization of valuable application insights is made possible.  Using the the Datadog HTTPS logging endpoint, the integration can quickly be setup through the [ngrok dashboard UI][2].
```

Comment:
Can you include the following in the overview? 

1. What does ngrok do in 1-2 sentence. (This was answered in the first paragraph)
2. What will customers get from this integration? (In term of value) This should be clear enough for a new customer to understand what this integration does. 
3. What types of data?